src/components/expression/expression-renderer.js

```diff
@@ -309,70 +454,60 @@ genotet.ExpressionRenderer.prototype.drawMatrixCells_ = function() {
       ])
       .range(genotet.data.redYellowScale);
   }
-  var transformLeft = 0;
-  var transformTop = this.conditionLabelHeight_ + this.profileHeight_;
-  if (this.data.options.showGeneLabels) {
-    transformLeft += this.LABEL_MARGIN_;
-  }
-  else if (this.data.options.showProfiles) {
-    transformLeft += this.DEFAULT_PROFILE_MARGIN;
-  }
-  if (this.data.options.showConditionLabels) {
-    transformTop += this.LABEL_MARGIN_;
-  }
 
   var heatmapRows = this.svgHeatmapContent_.selectAll('g')
     .data(heatmapData.values);
   heatmapRows.enter().append('g');
-  heatmapRows.attr('transform',
-    genotet.utils.getTransform([transformLeft, transformTop]));
   heatmapRows.exit().remove();
   var heatmapRects = heatmapRows.selectAll('rect').data(_.identity);
   heatmapRects.enter().append('rect');
   heatmapRects
-    .attr('width', cellWidth)
-    .attr('height', cellHeight)
+    .attr('width', this.cellWidth)
+    .attr('height', this.cellHeight)
     .attr('x', function(value, i) {
-      return i * cellWidth;
-    })
+      return i * this.cellWidth;
+    }.bind(this))
     .attr('y', function(value, i, j) {
-      return j * cellHeight;
-    })
+      return j * this.cellHeight;
+    }.bind(this))
     .style('stroke', colorScale)
     .style('fill', colorScale)
     .on('mouseover', function(value, i, j) {
       var hoverCell = d3.event.target;
-      var cell = this.cell_ = {
+      var cell = new genotet.ExpressionRenderer.Cell();
+      _(cell).extend({
```

Comment:
Umm if you are extending many attributes, probably a better way is to construct by params.
```js
genotet.ExpressionRenderer.Cell = function(params) {
  this.container = params.container;
  this.geneName = params.geneName;
  // ... and so on
}

// Here:
var cell = new genotet.ExpressionRenderer.Cell({
  container: hoverCell,
  geneName: heatmapData.geneNames[j],
  // ... and so on
});
```